content/shmem_iput.tex

```diff
@@ -70,8 +70,10 @@
     greater than or equal to \CONST{1}. The routines return when the data has
     been copied out of the \VAR{source} array on the local \ac{PE} but not
     necessarily before the data has been delivered to the remote data object.
-    If the context handle \VAR{ctx} does not correspond to a valid context,
-    the behavior is undefined.
+	The routines are considered complete after a subsequent call to
+	\FUNC{shmem\_quiet}.
+	If the context handle \VAR{ctx} does not correspond to a valid context,
+     the behavior is undefined.
 }
```

Comment:
(Notwithstanding the metadiscussion regarding memory model, needing definitions for completion, etc.)

These two lines at 75-76 are marked as a change when there doesn't look like there's any text that has changed. I think there's a whitespace indentation that is messing up the diff. Please fix so these lines don't appear in the git diff.